bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/constants/RuntimeConstants.java

```diff
@@ -56,12 +56,13 @@
     public static final String USER_HOME = "user.home";
     public static final String BALLERINA_HOME = "ballerina.home";
     public static final String BALLERINA_PACKAGE_PREFIX = "ballerina" + ORG_NAME_SEPARATOR;
-    public static final String BALLERINA_MAX_POOL_SIZE_ENV_VAR = "BALLERINA_MAX_POOL_SIZE";
```

Comment:
So with this change, we cannot set the pool size with env variable? There are some docs related to that. we might need to change those too.